CHANGES.txt

```diff
@@ -4,8 +4,15 @@ Changes
 1.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+* NOTE: this is the last version to officially support Python 2.
```

Comment:
Perhaps "to officially support Python 2 only", or do you really intend to go Python 3-only in the future?

Reply:
see comment below.